spec/system/superintendent/manager_register_superintendent_spec.rb

```diff
@@ -1,12 +1,50 @@
 require 'rails_helper'
 
 describe 'Manager register superintendent' do
-  it 'Successfully' do
+  it 'Successfully to act now' do
     condo = create :condo, name: 'Condomínio X'
     resident = create(:resident, :with_residence, full_name: 'Alvus Dumbledore', condo:)
     manager = create :manager
     travel_to '2024-07-21'.to_date
 
+    desactive_superintendent_job_spy = spy 'DesactiveSuperintendentJob'
+    stub_const 'DesactiveSuperintendentJob', desactive_superintendent_job_spy
+
+    login_as manager, scope: :manager
+    visit root_path
+    within 'nav' do
+      click_on id: 'side-menu'
+      click_on 'Gerenciar Usuários'
+      click_on 'Cadastrar Síndico'
+    end
+    within '#condoSelectPopupForSuperintendent' do
+      click_on 'Condomínio X'
+    end
+
+    fill_in 'Data de início', with: '2024-07-21'.to_date
+    fill_in 'Data de conclusão', with: '2025-07-25'.to_date
+    select 'Alvus Dumbledore', from: 'Morador'
+    click_on 'Enviar'
+
+    expect(page).to have_content 'Mandato de síndico cadastro com sucesso!'
+    expect(current_path).to eq condo_superintendent_path(condo, Superintendent.last)
+    expect(Superintendent.last.tenant).to eq resident
+    expect(Superintendent.last.in_action?).to eq true
+    expect(desactive_superintendent_job_spy).to have_received(:set).with({ wait_until: '2025-07-25'.to_datetime })
+  end
+
+  it 'Successfully to act on 2024-07-25' do
```

Comment:
Acho que ficaria mais claro o que o teste pretende testar se estivesse algo como: "successfully to act on future"